api.md

```diff
@@ -2253,23 +2253,25 @@ Get metadata and settings information for continuous aggregates.
 |`refresh_lag` | Amount by which the materialization for the continuous aggregate lags behind the current time |
 |`refresh_interval` | Interval between updates of the continuous aggregate materialization|
 |`max_interval_per_job` | Maximum amount of data processed by a materialization job in a single run|
+|`ignore_invalidation_older_than` | Do not update continous aggregate view for older updates|
 |`materialization_hypertable` | Name of the underlying materialization table|
 |`view_definition` | `SELECT` query for continuous aggregate view|
 
 #### Sample Usage
 ```sql
 select * from timescaledb_information.continuous_aggregates;
--[ RECORD 1 ]--------------+-------------------------------------------------
-view_name                  | contagg_view
-view_owner                 | postgres
-refresh_lag                | 2
-refresh_interval           | 12:00:00
-max_interval_per_job       | 20
-materialization_hypertable | _timescaledb_internal._materialized_hypertable_2
-view_definition            |  SELECT foo.a,                                  +
-                           |     count(foo.b) AS countb                      +
-                           |    FROM foo                                     +
-                           |   GROUP BY (time_bucket(1, foo.a)), foo.a;
+-[ RECORD 1 ]------------------+-------------------------------------------------
+view_name                      | contagg_view
+view_owner                     | postgres
+refresh_lag                    | 2
+refresh_interval               | 12:00:00
+ignore_invalidation_older_than | 1 month
```

Comment:
ignore_invalidation_older_than looks incorrect.

Reply:
In what sense? It cannot be set to `1 month` so I changed that to `7 days` (and verified that it can actually be set).

Reply:
refresh_lag says 2, so thought this is a cont. aggregate on an integer based hypertable. Is it?

Reply:
Right, this is what I get when I run the tutorial:
```
nyc_data=# SELECT * FROM timescaledb_information.continuous_aggregates;
-[ RECORD 1 ]------------------+----------------------------------------------------------------------------------------
view_name                      | cagg_rides_view
view_owner                     | postgres
refresh_lag                    | 02:00:00
refresh_interval               | 00:30:00
max_interval_per_job           | 20:00:00
ignore_invalidation_older_than | 106751991 days 04:00:54.775807
materialization_hypertable     | _timescaledb_internal._materialized_hypertable_2
view_definition                |  SELECT rides.vendor_id,                                                               +
                               |     time_bucket('01:00:00'::interval, rides.pickup_datetime) AS day,                   +
                               |     count(*) AS total_rides,                                                           +
                               |     avg(rides.fare_amount) AS avg_fare,                                                +
                               |     max(rides.trip_distance) AS max_trip_distance,                                     +
                               |     min(rides.trip_distance) AS min_trip_distance                                      +
                               |    FROM rides                                                                          +
                               |   GROUP BY rides.vendor_id, (time_bucket('01:00:00'::interval, rides.pickup_datetime));
```